.gitignore

```
@@ -13,3 +13,5 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+/.vs/ProjectSettings.json
```

Se for realmente adicionar esta pasta na lista de exclusão acho que seria melhor então excluir a pasta inteira
```suggestion
/.vs/
```

---

ok, de acordo.